data/856/791/95/85679195.geojson

```diff
@@ -18,6 +18,12 @@
     "label:eng_x_preferred_placetype":[
         "province"
     ],
+    "label:tur_x_preferred_longname":[
+        "Ayd\u0131n ili"
```

Comment:
Since this is native speaker in the home country, should the `region` labels longname be stored into an alternate instead?

Reply:
@nvkelso - Hmm... are you suggesting the properties should look like this?:

```
 "label:tur_x_preferred_longname":[
        "Ayd\u0131n"
 ],
 "label:tur_x_variant_longname":[
        "Ayd\u0131n ili"
 ],
```

If so, I understand the idea behind it, but I'll have to revisit many of the recent PRs and adjust accordingly.

Reply:
Yes, this point-of-view stuff can be confusing. Think of it this way... in whatever country we're in the `region` names are all pretty well known... but all the `county` names aren't so it's useful to include the placetype for the counties, but not for the regions. But for regions outside of the country it'd be useful.

Reply:
Yeah, that makes sense. I've been adding label values without "home region" consideration, so I'll be filing follow-ups in various repos.

Ill ping you in this issue once fixed.